{
  try {
    handleObjectDelete();
  }
 catch (  Exception e) {
    handleException(e);
  }
}
